### PR TITLE
Basic PR template to help guide verification process

### PR DESCRIPTION
**Pull request by jbarrett**
Cc: @nilnilnil @jdorweiler @russellholt 

Interested in:

- Is this useful?
- Should it contain more? Or less?

### CODE REVIEW

**Comment by jbarrett** [comment]
Do we also want an issue template?

I think on the whole the issue descriptions we get tend to be adequate, so leaning towards no.

File: PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+Reviewer notes :
+
+
+References issues / PRs:
+
+#
+
+Checklist :
+- [ ] Perl unit tests
+- [ ] Front end unit tests
```
Comment (MariagraziaAlastra) [inline PULL_REQUEST_TEMPLATE]: What about front end integration tests?

**Comment by MariagraziaAlastra** [comment]
@jbarrett Thanks, this looks helpful!
I agree about the issues being already well explained so far, so either no or yes, but extremely low priority, imho

**Comment by jdorweiler** [comment]
Are there any docs on how to run the tests?  It would be nice to link to those if we have them.

File: PULL_REQUEST_TEMPLATE (hunk 1 above)
Comment (jbarrett) [inline PULL_REQUEST_TEMPLATE]: Do we have both? Should I rename this entry or add one?

**Comment by jbarrett** [comment]
@jdorweiler Nothing specific - I think they would be so short we could inline them.

`$ dzil test --release --author && grunt test-all # or something`

File: PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,28 @@
+##### Description :
+
+
+##### Reviewer notes :
+
+
+##### References issues / PRs:
+
+#
```
Comment (GuiltyDolphin) [inline PULL_REQUEST_TEMPLATE]: Is this a place-holder for an issue reference?

File: PULL_REQUEST_TEMPLATE [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+##### Who should be informed of this change?
+
+
+##### Does this change have significant privacy, security, performance or deployment implications?
```
Comment (GuiltyDolphin) [inline PULL_REQUEST_TEMPLATE]: Maybe `Does this change have any`?

File: PULL_REQUEST_TEMPLATE (hunk 2 above)
Comment (jbarrett) [inline PULL_REQUEST_TEMPLATE]: Yep, I'm catering to the truly lazy!

File: PULL_REQUEST_TEMPLATE (hunk 3 above)
Comment (jbarrett) [inline PULL_REQUEST_TEMPLATE]: If it helps readability - don't know if it does, really.

Comment (MariagraziaAlastra) [inline PULL_REQUEST_TEMPLATE]: Either way it's fine to me

Comment (GuiltyDolphin) [inline PULL_REQUEST_TEMPLATE]: @jbarrett Yeah - either is fine; it doesn't change the meaning.

**Comment by MariagraziaAlastra** [comment]
@jbarrett Happy to merge here?

**Comment by jbarrett** [comment]
:+1: Thanks!

**Comment by MariagraziaAlastra** [comment]
Thank you @jbarrett! :+1:

**Pull request merged**